Tools/SandboxTest.ps1

```diff
@@ -171,7 +171,7 @@ $bootstrapSettingsContent['logging'] = @{level = 'verbose' }
 if ($EnableExperimentalFeatures) {
   $bootstrapSettingsContent['experimentalFeatures'] = @{
     dependencies    = $true

```

Comment:
Nit: Remove the extra space before the `=`. Looks like it used to be aligned with the line below, but it changed now